HW1/P4/P4.py

```diff
@@ -0,0 +1,35 @@
+# import findspark
+# findspark.init()
+import pyspark
+import numpy as np
+
+from P4_bfs import *
+
+
+# We lode the data and create the graph
+
+superheroes = sc.textFile('./P4/source.csv')
```

Comment:
Kind of a bizarre way to open the file using "./P4/source.csv".. I think this means python will only find the file if call from HW1.  Or does this work if you run the script from the command line while in HW1/P4/ ?